lib/medera/slack/event.ex

```diff
@@ -38,6 +44,8 @@ defmodule Medera.Slack.Event do
   defp event_type(%{type: "hello"}),           do: :hello
   # when a user is setting/unsetting 'away'
   defp event_type(%{type: "presence_change"}), do: :presence_change
+  # user opens a direct message channel
+  defp event_type(%{type: "im_open"}),         do: :im_open
```

Comment:
I was trying to whitelist them but I'm thinking about changing this a bit since each message type has a (potentially) different payload.

BTW in general `String.to_atom/1` is something to be careful about because atoms are never freed and therefore it can be a memory leak.  `String.to_existing_atom/1` is safer.